models/stg_amplitude__event.sql

```diff
@@ -71,6 +72,9 @@ final as (
         version_name,
         _fivetran_synced
     from fields
+
+    where cast({{ dbt.date_trunc('day', 'event_time') }} as date) >= {{ "cast('" ~ var('date_range_start',  '2020-01-01') ~ "' as date)" }} -- filter to records past a specific date
```

Comment:
This actually makes me think we should now consider this a as a breaking change. Otherwise, we will start filtering customers data out of the blue in a patch release. Following this, let's treat this update as a breaking change.

Reply:
Additionally, should we consider date_range_end here as well? Not sure if that makes sense, but wanted to pose the idea.

Reply:
Those are good points, will treat as breaking change and adjust accordingly!

And yes, I figured that we didn't need date_range_end since customers likely care to see the most recent data. But it won't hurt to add it in and just

Reply:
Just to be safe, let's do current timestamp plus a month. Just in case there is some weird timing issue where dates are coming in from a day or so in the future.